docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+description: Getting Started with Hasura & MS SQL Server
+keywords:
+  - hasura
+  - docs
+  - databases
+  - ms sql
+  - sql server
+---
+
+# Getting started with MS SQL Server
+
+To try Hasura with SQL Server, you'll need your own new or existing SQL
+Server database.
+
+Here are 2 ways you can get started with Hasura:
+
+1.  [Hasura Cloud](docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/cloud.mdx) : You'll need to be
+    able to access your SQL Server database from Hasura Cloud.
+2.  [Docker](docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/docker.mdx): Run Hasura with Docker and
+    then connect your SQL Server database to Hasura.
+
+
+- [Hasura Cloud](docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/cloud.mdx)
+- [Docker](docs-new/docs/graphql/core/databases/ms-sql-server/getting-started/docker.mdx)
```

Comment:
Please correct the root relative links - they should start with `/graphql/.....` as discussed.